### DISCUSSION TITLE
Welcome to Hector9000 Discussions!

### DISCUSSION THREAD

**Discussion by DevTown** [H3c702/Hector9000#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ojaatka** [comment]
Hi Hector Crew,

I have now gone almost 50 years without a cocktail machine, this must now be changed.
Have the hardware under control, the software is running as well. 

Instead of a touchscreen, I would rather control the Hector 9000 with an rotary encoder and 1 to 2 buttons. 

Unfortunately, I have no idea how best to realize this.

Maybe someone has a hint for me?  
Maybe there is a simple way?

bye, Olivier
